integration/substrate/asserts.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('Deploy asserts contract and test', () => {
         expect(res0.output?.toJSON()).toEqual(1);
 
         let res1 = await query(conn, alice, contract, "testAssertRpc");
-        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "7" } } });
+        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "8" } } });
```

Comment:
Why did you change this file?
It seems unrelated to this PR.

Reply:
The substrate tests don't work anymore unless I change this.

Reply:
@xermicus do you know anything about this?

Reply:
Our CI uses some  `latest` docker image for the substrate contracts node. There was an update published a couple hours ago which is causing the error.

Before the update, the `contracts` pallet was the 7th module in the pallet, but now it is the 8th. The but the actual error is still the same (`ContractTrapped`, which is granted, not very obvious here), so this change is fine. Unless you want me to open a dedicated PR to fix this

Reply:
#1146 will fix this, sorry